region/crateria/central.json

```diff
@@ -2117,29 +2248,99 @@
           "to": [
             {
               "id": 3,
-              "requires": [
-                {"canShineCharge": {
-                  "usedTiles": 31,
-                  "shinesparkFrames": 130,
-                  "openEnd": 2
-                }}
-              ]
+              "strats": [
+                {
+                  "name": "Vanilla Strat",
+                  "notable": false,
+                  "requires": [
+                    {"canShineCharge": {
+                      "usedTiles": 31,
+                      "shinesparkFrames": 130,
+                      "openEnd": 2
+                    }},
+                    "canUseFrozenEnemies"
+                  ]
+                },
+                {
+                  "name": "Kill the Boyons",
```

Comment:
I find this one hard to get anything useful out of without a shortcharge, compared to bluesuit jump

Reply:
Hm... you're right. I feel like I could just make it notable as an alternate version, but that feels pointless.

I'll change it into a 1-tap, that'll make it a meaningful alternative.